.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+### Description
+
+**Summary:**
+
+<!-- Provide a brief overview of what this PR does or aims to achieve. -->
+
+**Related Issue:**
+
+<!-- Link to the GitHub issue that this PR addresses, if applicable. -->
+
+### Testing
+
+<!-- Describe the steps to test the changes made in this PR. Include details
+about the test environment, any specific configurations or data required, and
+the expected outcomes. -->
+
+### Screenshots/GIFs (if applicable)
+
+<!-- Include any screenshots or GIFs that help visualize the changes made,
+especially for UI-related changes. -->
+
+### Checklist
+
+-   [ ] My code follows the project's coding standards.
+-   [ ] I have written unit tests (if applicable) and they pass successfully.
+-   [ ] I have updated the documentation (if applicable).
+-   [ ] All existing tests and CI/CD pipelines are passing.
```

Comment:
Just wondering if we really need checklist items to say that the tests are passing? CI will catch this. I'd be happy with just one item: `I have written unit tests (if applicable)`

Reply:
True, I updated it.